.github/workflows/seed-data.yml

```diff
@@ -23,35 +23,26 @@ on:
           - production
       core-image-tag:
         description: Core DockerHub image tag
+        required: true
 jobs:
   seed-data:
     environment: ${{ github.event.inputs.environment }}
     runs-on: ubuntu-24.04
     timeout-minutes: 60
+    
     steps:
-      - name: Clone core
-        uses: actions/checkout@v3
+      - name: Run the data-seeder container
+        uses: addnab/docker-run-action@v3
         with:
-          fetch-depth: 0
-          repository: 'opencrvs/opencrvs-core'
-
-      - name: Set CORE_VERSION from inputs
-        if: ${{ github.event.inputs.core-image-tag }}
-        run: git checkout ${{ github.event.inputs.core-image-tag }}
-
-      - name: Use Node.js from .nvmrc
-        uses: actions/setup-node@v4
-        with:
-          node-version-file: .nvmrc
-
-      - name: Install dependencies
-        run: yarn install
+           username: ${{ github.actor }}
+           password: ${{ secrets.GITHUB_TOKEN }}
+           registry: gcr.io
+           image: ocrvs-data-seeder:${{ github.event.inputs.core-image-tag }}
+           run: yarn seed:prod
```

Comment:
Will the image be built in a way such that it won't run any command by default? That could be a good option actually